lib/global.dart

```diff
@@ -5,3 +5,7 @@ disabilityMode is to save the state of the toggle from the app drawer.
 it should be removed once a config file is used to save the user preferences
```

Comment:
We will need to replace the globals with a DirectionsNotifier class

Reply:
You can use the global or change it to a full class that deals with TravelModeNotifier, if you deem more beneficial. That's an implementation detail for those implementing the directions 👍